pom.xml

```diff
@@ -30,6 +30,9 @@
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>4.18.1</version>
+      <groupId>org.seleniumhq.selenium</groupId>
```

Comment:
**Issue**: Duplicate dependency declaration for 'selenium-java' in the 'pom.xml' file. This can lead to confusion and potential build issues if different versions are specified in each declaration. <br> **Fix**: Remove the duplicate dependency declaration to maintain a clean and error-free pom configuration. <br> **Code Suggestion**: 
 ```
 @@ -33,3 +33,0 @@
 -      <groupId>org.seleniumhq.selenium</groupId>
 -      <artifactId>selenium-java</artifactId>
 -      <version>4.18.1</version>
 ```